packages/charts/src/hooks/use-distributed-chart-tooltip-settings.ts

```diff
@@ -0,0 +1,84 @@
+import {
+  formatValue as valueFormatter,
+  getFromPath,
+  getKeysDifference,
+  ScaleSettings,
+  TooltipFormatter,
+  transformToPercent,
+} from '@keen.io/charts-utils';
+
+import { DataSelector } from '../types';
+import { getLabel } from '../components/distributed-chart-tooltip';
+
+const useDistributedChartTooltipSettings = (
+  data: Record<string, any>[],
+  keys: string[],
+  disabledKeys: string[],
+  selectors: { selector: DataSelector; color: string }[],
+  isPercentage: boolean,
+  labelSelector: string,
+  maxWidth?: number,
```

Comment:
Based on code the `maxWidth` is obsolete here. 

Also I suggest to change `interface` to use `object` instead of inline arguments. Especially having in mind that this `hook` accepts a lot of arguments and passing `object` will be much easier. What do You think ?